include/xten/Transform/PassesTorch.td

```diff
@@ -0,0 +1,34 @@
+//===- Passes.td -------------------------------------------*- tablegen -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+
+// (c) Copyright 2021 Xilinx, Inc. All Rights reserved.
+// (c) Copyright 2022 - 2024 Advanced Micro Devices, Inc. All Rights reserved.
+
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ATEN_TRANSFORM_PASSES_TORCH
+#define ATEN_TRANSFORM_PASSES_TORCH
+
+include "mlir/Pass/PassBase.td"
+
```

Comment:
Haven't you deleted all these passes (Transforms/ATenDialectOpStats.cpp etc.)? Do we still need the table gen file?